synfig-studio/src/gui/app.cpp

```diff
@@ -1350,8 +1349,9 @@ App::get_default_accel_map()
 
 /* === M E T H O D S ======================================================= */
 
-Glib::RefPtr<App> App::create()
+Glib::RefPtr<App> App::create(const synfig::String& basepath)
 {
+	app_base_path_=etl::dirname(basepath);
 	return Glib::RefPtr<App>(new App());
```

Comment:
You don't have to make this cast here.
Your return type already knows how to encapsulate the pointer into it. :)

Reply:
Understood ! ✅🙂